tethys_cli/install_commands.py

```diff
@@ -855,9 +856,11 @@ def install_command(args):
         if validate_schema("post", install_options):
             write_msg("Running post installation tasks...")
             for post in install_options["post"]:
-                path_to_post = file_path.resolve().parent / post
+                command = file_path.resolve().parent / post
```

Comment:
I would prefer this to be called something like `script` rather than `command`. Since it is a path to a file I think `command` is somewhat missleading.

Reply:
Yeah, I'm honestly not sure why I even switch to `command` from `path_to_post`. I think I'll just revert to `path_to_post`.